build/devenv/services/aggregator.go

```diff
@@ -286,9 +285,21 @@ func NewAggregator(in *AggregatorInput) (*AggregatorOutput, error) {
 	}
 
 	if in.SourceCodePath != "" {
+		cwd, err := os.Getwd()
+		if err != nil {
+			return nil, fmt.Errorf("failed to get working directory: %w", err)
+		}
+
 		req.Mounts = testcontainers.Mounts()
 		req.Mounts = append(req.Mounts, GoSourcePathMounts(in.RootPath, AppPathInsideContainer)...)
 		req.Mounts = append(req.Mounts, GoCacheMounts()...)
+
+		// TODO: Generate config file, write it to a local path, and mount it here.
+		req.Mounts = append(req.Mounts, testcontainers.BindMount( //nolint:staticcheck // we're still using it...
+			filepath.Join(cwd, "configs", "aggregator", in.CommitteeName, "aggregator.toml"),
+			//aggregator.DefaultConfigFile,
+			"/etc/config.toml",
+		))
```

Comment:
@skudasov the main change in this PR. Inject config into the docker container. I'll need to repeat this for all services.